apps/chat/src/utils/server/api.ts

```diff
@@ -107,10 +107,14 @@ export class ApiUtils {
     }).pipe(
       switchMap((response) => {
         if (!response.ok) {
-          return throwError(() => new Error(response.status + ''));
+          return from(response.text()).pipe(
+            switchMap((errorMessage) =>
+              throwError(() => new Error(errorMessage)),
+            ),
+          );
         }
 
-        return from(response.json());
+        return response.json();
```

Comment:
Why did you remove `from` here?

Reply:
Was trying another way of handling error and forgot to revert the changes